backend/postgres/schema.sql

```diff
@@ -0,0 +1,57 @@
+CREATE SCHEMA IF NOT EXISTS durabletask;
+
+CREATE TABLE IF NOT EXISTS durabletask.Instances (
+    InstanceID VARCHAR PRIMARY KEY NOT NULL,
```

Comment:
Since these are UUIDs, consider using the native UUID type for Postgres. It's more efficient as it stores the UUID as a binary sequence (internally). This is especially helpful since the UUID is used as primary key, so it's in many indexes

Reply:
We don't restrict instance IDs to be UUIDs. This is just our convention for defaults. Many users, however, will desire to provide their own instance IDs to match some business entity or to implement the singleton pattern.

Reply:
Makes sense